src/components/settings/settings-access-status-types/settings-access-status-types.js

```diff
@@ -0,0 +1,163 @@
+  };
+
+  const descriptionValidation = value => {
+    if (value && value.length > 150) {
+      return <FormattedMessage
+        id="ui-eholdings.settings.accessStatusTypes.validation"
+        values={{ limit: 150 }}
+      />;
+    } else {
+      return null;
+    }
+  };
+
+  return (
+    <Pane
+      paneTitle={<FormattedMessage id="ui-eholdings.settings.accessStatusTypes" />}
+      data-test-settings-access-status-types
+      defaultWidth="fill"
+      firstMenu={(
+        <FormattedMessage id="ui-eholdings.settings.goBackToEholdings">
+          {ariaLabel => (
+            <PaneCloseLink
+              ariaLabel={ariaLabel}
+              to="/settings/eholdings"
+            />
+          )}
+        </FormattedMessage>
+      )}
+    >
+      <IntlConsumer>
+        {intl => (
+          <EditableList
+            actionProps={accessTypesData && accessTypesData.length >= 15 ? { create: () => ({ disabled: true }) } : {}}
+            columnMapping={{
+              name: intl.formatMessage({ id: 'ui-eholdings.settings.accessStatusTypes.type' }),
+              description: intl.formatMessage({ id: 'ui-eholdings.settings.accessStatusTypes.description' }),
+              lastUpdated: intl.formatMessage({ id: 'ui-eholdings.settings.accessStatusTypes.lastUpdated' }),
+              records: intl.formatMessage({ id: 'ui-eholdings.settings.accessStatusTypes.records' }),
+            }}
+            contentData={accessTypesData}
+            createButtonLabel={intl.formatMessage({ id: 'ui-eholdings.new' })}
+            fieldComponents={{
+              name: item => renderField(item, nameValidation),
+              description: item => renderField(item, descriptionValidation),
+            }}
+            formatter={formatter}
+            label={intl.formatMessage({ id: 'ui-eholdings.settings.accessStatusTypes' })}
+            onCreate={onCreateItem}
+            onDelete={onDelete}
+            onUpdate={onUpdate}
+            readOnlyFields={['lastUpdated', 'records']}
+            visibleFields={['name', 'description', 'lastUpdated', 'records']}
+          />
+        )}
+      </IntlConsumer>
+    </Pane>
+  );
+};
+
+SettingsAccessStatusTypes.propTypes = {
+  accessTypesData: PropTypes.arrayOf(PropTypes.shape({
+    attributes: PropTypes.shape({
+      description: PropTypes.string,
+      name: PropTypes.string.isRequired,
+    }).isRequired,
+    creator: PropTypes.objectOf(PropTypes.string),
+    id: PropTypes.string.isRequired,
+    metadata: PropTypes.shape({
+      createdByUserId: PropTypes.string.isRequired,
+      createdByUsername: PropTypes.string.isRequired,
+      createdDate: PropTypes.string.isRequired,
+      updatedByUserId: PropTypes.string,
+      updatedDate: PropTypes.string,
+    }),
+    type: PropTypes.string.isRequired,
+    updater: PropTypes.objectOf(PropTypes.string),
+  })),
+  onCreate: PropTypes.func.isRequired,
+  onDelete: PropTypes.func.isRequired,
+  onUpdate: PropTypes.func.isRequired,
+};
```

Comment:
If you have `props `which are not required, I would recommend to add them to `defaultProps`